### ISSUE TITLE
cant get into to work on mac os

### ISSUE THREAD

**Issue by moshetanzer** [VikParuchuri/marker#365]
using pip and python 3.9 in venv

get the following error
````
TypeError: unsupported operand type(s) for |: '_GenericAlias' and 'NoneType'
````

**Comment by LOEE-PYTHON** [comment]
me too

**Comment by moshetanzer** [comment]
This is solved with using specifcaily python 3.10 on latest version. Unfortunatly there are tons of other bugs when trying to use this libary!
